### PR TITLE
Chapter 02 and 03

### PR DESCRIPTION
**Pull request by jeffkala**

### CODE REVIEW

File: chapters/ch02/ansible.cfg
Comment (chadell) [inline chapters/ch02/ansible.cfg:1]: empty?

File: chapters/ch02/configs/ceos-01.cfg
Comment (chadell) [inline chapters/ch02/configs/ceos-01.cfg:1]: should we leave this out? just the folder?

File: chapters/ch03/fact_cache/ceos-01
Comment (chadell) [inline chapters/ch03/fact_cache/ceos-01:1]: should we leave this out?

File: chapters/ch03/roles/basic_config/README.md
Comment (chadell) [inline chapters/ch03/roles/basic_config/README.md:1]: do we need to add all the boilderplate in the repo?

**Review by chadell — APPROVED** [review]
LGTM, but I think that is better to leave the results of commands out of the repo

**Comment by jeffkala** [comment]
fyi @chadell I don't have privs to merge this PR

**Comment by chadell** [comment]
fixed

Missatge de Jeff Kala ***@***.***> del dia ds., 14 de des.
2024 a les 5:30:

**Pull request merged**